src/basic/time-util.c

```diff
@@ -267,7 +267,7 @@ static char *format_timestamp_internal(
 
         assert(buf);
 
-        if (l <
+        if (l < (size_t)
```

Comment:
hmm, i wonder if it wouldn't be better to cast the whole addition instead of just the first value of it

Reply:
OK. Done.